HISTORY.md

```diff
@@ -4,6 +4,9 @@
 * Added values to `TraceFilterType`: `kTraceFilterIteratorSeek`, `kTraceFilterIteratorSeekForPrev`, and `kTraceFilterMultiGet`. They can be set in `TraceOptions` to filter out the operation types after which they are named.
 * Added `TraceOptions::preserve_write_order`. When enabled it  guarantees write records are traced in the same order they are logged to WAL and applied to the DB. By default it is disabled (false) to match the legacy behavior and prevent regression.
 
+### Performance Improvements
+* Released the cache reservation of the hash entries associated with the Ribbon Filter that has falled back to Bloom Filter earlier for `BlockBasedTableOptions::reserve_table_builder_memory = true`.
```

Comment:
I think this change is such a small corner case, it is not worth mentioning in HISTORY.md. (People have limited attention span and we have evidence of people not paying attention to release notes, so we should be judicious in what we include by making them "worth reading.") Thanks